export_kakuma_advert_contacts.py

```diff
@@ -154,7 +182,7 @@
         }
 
     log.warning(f"Exporting {len(advert_contacts)} contacts to {contacts_csv_path}")
-    with open(contacts_csv_path, "w") as f:
+    with open(f'{contacts_csv_path}.csv', "w") as f:
```

Comment:
```suggestion
    with open(contacts_csv_path, "w") as f:
```
For consistency with the rest of the pipeline